docs/guide/tools/geometry.md

```diff
@@ -0,0 +1,37 @@
+# Geometry tools
+
+In the latest development version, there is a setsquare tool that can be used for measuring lengths and angles and for
+geometric constructions.
+
+![Setsquare-Overview](../../img/guide/setsquare_overview.png)
+
+A compass wheel is in work as well.
+
+## Setsquare handling
+
+- **Switch on/off** either from menu `Edit > Setsquare` or via the toolbar (setsquare symbol)
```

Comment:
The current list style is hard to read in my opinion. I think this would be easier to read if separated into subsections, e.g.:

```markdown

## Setsquare

Overview of what the tool is for and how to set it up.

### Keyboard Controls

Description...

### Touch Controls

Description...

### Interactions with other tools

Description of how it works with pen/highlighter
```

Reply:
Thanks for the suggestion. I tried to improve the structure of this content. There is space for more improvement, I think, but it's already quite a bit more readable, I guess.